src/common/layout/Header.js

```diff
@@ -190,7 +195,7 @@ const renderLoggedOut = (
         onClick={() => push('/sign-in')}
         type="secondary"
       >
-        Sign&nbsp;In
+        <Trans t={__('smartfaq.header.sign_in')} />
```

Comment:
Shouldn't this be
```jsx
<Trans html t={__('smartfaq.header.sign_in')} />
```

Otherwise how would the non-breaking space be rendered?

Reply:
I thought it was already added in my last commit, don't know what happened here 🙀